netbout-client/pom.xml

```diff
@@ -180,7 +180,12 @@
                         <artifactId>qulice-maven-plugin</artifactId>
                         <configuration>
                             <excludes>
-                                <exclude>findbugs:.*</exclude>
+                                <!--
+                                @todo #911 Fix findbugs errors in client
```

Comment:
@exper0  the puzzle description should contains at least 20 words, also you have to use format like this
`@todo #911 :30min Fix findbugs errors in client`
I wonder why Travis's build succeeded